### DISCUSSION TITLE
Ended 3 v2 w/ sprite pro extruder issues

### DISCUSSION THREAD

**Discussion by LuckyLiem** [mriscoc/Ender3V2S1#840] [Q&A]
I recently got a sprite extruder pro for my ender 3 v2 and followed the instructions exactly. But when is try to use it on this firmware it doesn't use the car touch I have on it properly and when it tries to use the cr touch on an edge the cr touch will be off the bed and when it goes down to level, it goes down infinitely on the z axis. Does this have anything to do with having to change the E steps value to 424.9?

**Comment by mriscoc** [comment]
Please read the firmware documentation, Wiki and release notes.
https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Comment by I-wish-2** [comment]
Just a thought,
As I have also installed the Sprite Pro on one of my E3 Pros,  When you first power up the printer before any home commands does the CR Touch go through its initialization cycle ok?  I found the ribbon cable wiring on my Sprite was not correct at the 5 pin plug to the main motherboard causing the CR Touch to only show it's red led indication of a function or wiring problem. 
I hope that might help some what.
D
